client/todo-list.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+export default function TodoList({ todos }) {
+  return (
+    <div className='row'>
+      <div className='col xl8 l8 m12 s12 push-xl2 push-l2'>
+        <div className='row'>
+          { todos.map((todo, index) => ListItem(todo, index)) }
+        </div>
+      </div>
+    </div>
+  )
+}
+
+function ListItem({ task, due }, index) {
```

Comment:
A subtle point: Capitalizing a function name and returning JSX implies that this function is a Function Component, but it isn't. Components are always passed a `props` object and that is not what this function expects. I recommend renaming it to `renderListItem`.

Reply:
Gotcha, that makes sense. Definitely want to make sure that I'm not mis-communicating here in the future. Thank you for the clarification.